### ISSUE TITLE
Upstreaming to kaitai_struct_formats

### ISSUE THREAD

**Issue by KOLANICH** [Jakeler/bms-parser#2]
https://github.com/kaitai-io/kaitai_struct_formats/

**Comment by Jakeler** [comment]
Yes, I think that is a good idea. Though I am not sure in which category/folder it would fit and under what name. Is `bms_protocol` or `battery_management_system` clear enough? How would you call and categorize it? @KOLANICH

**Comment by KOLANICH** [comment]
1. we have a cathegory `hardware`
2. I have plans to bring some specs related to [Smart Battery](http://sbs-forum.org/specs/). So we need a `battery/bms` dir.
3. Since there is no namespaces (all generated files are placed flatly) currently, the specs names should be global. So I guess `hardware/battery/bms/battery_management_system_protocol.ksy`

**Comment by Jakeler** [comment]
Thanks, see: https://github.com/kaitai-io/kaitai_struct_formats/pull/410

**Comment by eiman5899** [comment]
Hye just wanted to ask, already ask at your email. how do you connect the bms? are you using the bluetooth or the uart usb?

**Comment by Jakeler** [comment]
@eiman5899 Does not relly make a difference as the script just reads/writes to a device file. Can be either a direct cable connection with an UART/USB adapter. Or wirelessly connected 2 bluetooth module ending again in the UART adapter. 
Or (what I always use these days) direct wireless connection to the bluetooth dongle on the bms with [ble-serial](https://github.com/Jakeler/ble-serial), it provides a virtual serial port `/tmp/ttyBLE` which can be used like a real one.
